src/components/Person/Person.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Person.scss';
+
+function Person({ personalInfo }) {
+  const {
+    name,
+    age,
+    partnerName,
+    isMarried,
+    sex,
+  } = personalInfo;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is {name}
+      </h2>
+      <p className="Person__age">
+        {age ? (
+          `I am ${age}`
+        ) : null}
+      </p>
+      {isMarried ? (
+        <p className="Person__partner">
+          {sex === 'm' ? (
+            `My wife's name is ${partnerName}`
+          ) : (
+            `My husband's name is ${partnerName}`
+          )}
+        </p>
+      ) : 'I am not married'}
```

Comment:
You can create sth like: 
`const partner = sex === 'm' ? 'wife' : 'husband';` 
This way you can use only one turnery operator and  put them in `<p></p>`. This way if married it's in p tag, if not there is no tag.